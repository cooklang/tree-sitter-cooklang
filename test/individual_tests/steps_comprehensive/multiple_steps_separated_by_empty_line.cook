First step here.

Second step starts after empty line.