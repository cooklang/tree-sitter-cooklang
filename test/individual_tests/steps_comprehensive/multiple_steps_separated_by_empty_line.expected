(recipe
  (step
    (plain_text))
  (step
    (plain_text)))